openerp/addons/base/migrations/9.0.1.3/post-migration.py

```diff
@@ -83,6 +84,18 @@ def rename_your_company(cr):
         """)
 
 
+def update_res_currency(cr):
+    openupgrade.logged_query(cr, """
+        DELETE FROM ir_rule
+        WHERE model_id IN (
+            SELECT id from ir_model
+            WHERE model = 'res.currency'
+        )
+        AND domain_force like '%%company_id%%'
+        """)
```

Comment:
why would we want to do that? Records with an XMLID will be deleted at the end of a successful migration anyways, and the migration itself runs as `SUPERUSER_ID`, so they won't get in the way